specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHubCluster.json

```diff
@@ -0,0 +1,606 @@
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Clusters returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/EHClusterListResult"
+            }
+          },
+          "default": {
+            "description": "Event Hub Cluster error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}": {
+      "get": {
+        "tags": [
+          "Event Hubs Cluster"
+        ],
+        "operationId": "Clusters_Get",
+        "x-ms-examples": {
+          "ClusterGet": {
+            "$ref": "./examples/Clusters/EHClusterGet.json"
+          }
+        },
+        "description": "Gets the resource description of the specified Event Hubs Cluster.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster resource description returned successfully.",
+            "schema": {
+              "$ref": "#/definitions/EHCluster"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response describing why the cluster description get operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Event Hubs Cluster"
+        ],
+        "operationId": "Clusters_Patch",
+        "x-ms-examples": {
+          "ClusterPatch": {
+            "$ref": "./examples/Clusters/EHClusterPatch.json"
+          }
+        },
+        "description": "Modifies mutable properties on the Event Hubs Cluster. This operation is idempotent.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EHCluster"
```

Comment:
Are all properties updatable via the PATCH

Reply:
Actually all RP specific properties are marked read-only. So only tags can be updated?

Reply:
Yes, for now only tags can be updated. We will open up additional properties in subsequent API versions.